### PR TITLE
stability test: async emit annotations

### PR DESCRIPTION
**Pull request by weekface**
### What problem does this PR solve? 

This pr fixes stability tests:
* async emit annotations
* operator image pull policy to always
* increase tidbcluster memory and cpu
* update to v2.1.7
* fix truncate sst case
* default concurrency to 12
* run stability test forever

### Check List 

Tests 

 - E2E test
 - Stability test

Code changes

 - Has Go code change

Side effects

Related changes

### CODE REVIEW

File: tests/actions.go [hunk 1]
```diff
@@ -1497,7 +1532,7 @@ func (oa *operatorActions) CheckAdHocBackup(info *TidbClusterConfig) error {
 }
 
 func (oa *operatorActions) Restore(from *TidbClusterConfig, to *TidbClusterConfig) error {
-	oa.emitEvent(to, fmt.Sprintf("RestoreBackup: source: %s", from.ClusterName))
+	oa.EmitEvent(from, fmt.Sprintf("RestoreBackup: source: %s", from.ClusterName))
```
Comment (xiaojingchen) [inline tests/actions.go]:
```suggestion
	oa.EmitEvent(from, fmt.Sprintf("RestoreBackup: target: %s", to.ClusterName))
	oa.EmitEvent(to, fmt.Sprintf("RestoreBackup: source: %s", from.ClusterName))
```

File: tests/actions.go [hunk 2]
```diff
@@ -531,9 +531,10 @@ func (oa *operatorActions) CheckTidbClusterStatus(info *TidbClusterConfig) error
 		}
 
 		glog.V(4).Infof("check tidb cluster begin metaSyncFn")
-		if b, err := oa.metaSyncFn(tc); err != nil {
-			return false, err
-		} else if !b && err == nil {
+		if b, err := oa.metaSyncFn(tc); !b && err == nil {
+			return false, nil
+		} else if err != nil {
+			glog.Error(err)
```
Comment (aylei) [inline tests/actions.go]: could you document why this error can be omitted?

Comment (weekface) [reply tests/actions.go]: other sync functions log the error [immediately](https://github.com/pingcap/tidb-operator/blob/31db0ea157d05e0e62d32626b2384cd33cbec772/tests/actions.go#L1064). `metaSyncFn ` return these errors and log them outside of the method.
these may be different coding styles.

**Pull request merged**